examples/models/llama/export_llama_lib.py

```diff
@@ -114,11 +118,11 @@ def build_model(
     else:
         output_dir_path = "."
 
-    argString = f"--checkpoint par:{modelname}_ckpt.pt --params par:{modelname}_params.json {extra_opts} --output-dir {output_dir_path}"
+    argString = f"--model {modelname} --checkpoint par:{modelname}_ckpt.pt --params par:{modelname}_params.json {extra_opts} --output-dir {output_dir_path}"
```

Comment:
it will not work here because currently `modelname` is set to `model`, for example, https://github.com/pytorch/executorch/blob/main/examples/models/llama/llama_test.py#L15-L23, while you expect modelname to be one of the `["llama2", "llama3", "llama3_1", "llama3_2"]`